src/expressions.js

```diff
@@ -183,11 +183,11 @@ var _ = Mavo.Expressions = $.Class({
 	},
 
 	extract: function(node, attribute, path, syntax = Mavo.Expression.Syntax.default) {
-		if (attribute && _.skip.indexOf(attribute.name) > -1) {
+		if (_.skip.includes(attribute?.name)) {
 			return;
 		}
 
-		if (attribute && _.directives.indexOf(attribute.name) > -1 ||
+		if (_.directives.includes(attribute?.name) || attribute?.name?.startsWith("mv-attr-") ||
```

Comment:
I wonder if we can do something less ad hoc here. E.g. make `_.directives` a list of strings and regexes, then check like:

```js
let attributeName = attribute?.name;
if (_.directives.some(d => d.test?.(attributeName) || d === attributeName)) || ...
```

Reply:
Great idea! Done.